vumi/scripts/vumi_model_migrator.py

```diff
@@ -97,7 +108,8 @@ def migrate_key(self, key, dry_run):
         try:
             obj = yield self.model.load(key)
             if obj is not None:
-                if obj.was_migrated and not dry_run:
+                should_save = yield self.post_migrate_function(obj)
+                if should_save and not dry_run:
```

Comment:
The default post-migrate function returns `obj.was_migrated`. If you provide your own post-migrate function, you can check `was_migrated` yourself if you care about it.